testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/search/search_basic.t.js

```diff
@@ -0,0 +1,152 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/**
+ * Test indexing & basic searching of indexed components.
+ */
+StartTest(function(t) {
+
+  var waitForStoreToLoad = {
+    waitFor: 'storesToLoad', args: function() {
+      return t.cq1('nx-coreui-search-result-list').getStore();
+    }
+  };
+
+  t.describe('Basic Indexing and Searching', function(t) {
+    t.it('Keyword search finds maven components', function(t) {
+      t.chain(
+          t.openPageAsAdmin('browse/search'),
+          function(next) {
```

Comment:
openPageAsAdmin() call now includes a waitForAnimations() so should be able to do away with all of the followup calls in this test